src/Enpress/Models/Post.php

```diff
@@ -300,5 +300,14 @@ public static function root($name)
             ->where('post_parent', 0)
             ->firstOrFail();
     }
+    
+    public function latestRevision()
+	{
```

Comment:
Is it the github web ui or is this indentation out of whack?

Reply:
Ergh web UI, I was being lazy.